kernel/core/net/net_loopback.c

```
@@ -0,0 +1,279 @@
+    tcp_reply[1] = tcp_syn[3];
+    tcp_reply[2] = tcp_syn[0];
+    tcp_reply[3] = tcp_syn[1];
+    tcp_reply[4] = 0;
+    tcp_reply[5] = 0;
+    tcp_reply[6] = 0;
+    tcp_reply[7] = 0;
+    tcp_reply[8] = (uint8_t)((seq >> 24) & 0xff);
+    tcp_reply[9] = (uint8_t)((seq >> 16) & 0xff);
+    tcp_reply[10] = (uint8_t)((seq >> 8) & 0xff);
+    tcp_reply[11] = (uint8_t)(seq & 0xff);
+    tcp_reply[12] = (uint8_t)(5u << 4);
```

Clang (14.0.6)

[warning] integer literal has suffix 'u', which is not uppercase

(readability-uppercase-literal-suffix)